diff --git a/node_modules/@khanacademy/flow-to-ts/src/transform.js b/node_modules/@khanacademy/flow-to-ts/src/transform.js
index 8ab9470..134b84a 100644
--- a/node_modules/@khanacademy/flow-to-ts/src/transform.js
+++ b/node_modules/@khanacademy/flow-to-ts/src/transform.js
@@ -412,6 +412,20 @@ const transform = {
             typeParameters
           )
         );
+      } else if (typeName.name === "Object") {
+        path.replaceWith(t.tsAnyKeyword());
+      } else if (typeName.name === "Function") {
+        const functionType = t.tsFunctionType(
+          null,
+          [{
+            type: "RestElement",
+            argument: t.identifier("args"),
+            decorators: null,
+            typeAnnotation: t.tsTypeAnnotation(t.tsAnyKeyword()),
+          }],
+          t.tsTypeAnnotation(t.tsAnyKeyword())
+        );
+        path.replaceWith(functionType);
       } else {
         path.replaceWith(t.tsTypeReference(typeName, typeParameters));
       }
@@ -566,6 +580,14 @@ const transform = {
       }
     }
   },
+  // TODO: Should this be replaced with a nominal type?
+  OpaqueType: {
+    exit(path) {
+      const { id, impltype, typeParameters } = path.node;
+
+      path.replaceWith(t.tsTypeAliasDeclaration(id, typeParameters, impltype));
+    }
+  },
   TypeAlias: {
     exit(path) {
       const { id, typeParameters, right } = path.node;
